README.md

```diff
@@ -57,6 +58,18 @@ sudo npm install -g svgo
 sudo apt-get install gifsicle
 ```
 
+Mozjpeg for Ubuntu:
+```bash
+sudo apt-get install autoconf automake libtool nasm make pkg-config git
```

Comment:
You could ship a statically built binary for the OS'es you support but that is an extra maintenance burden you might not want